stages/stage_2/defaults.py

```diff
@@ -10,11 +10,11 @@
 compiled and quoted, so it can be passed as an argument
 to a compiled clvm program.
 
-EG: (function (+ 20 @)) should return (+ (q 20) 1) when run.
-Thus (opt (com (q (function (+ 20 @)))))
-should return (q (+ (q 20) 1))
+EG: (function (+ 20 @)) should return (+ (q . 20) 1) when run.
+Thus (opt (com (q . (function (+ 20 @)))))
```

Comment:
This should be rebased on `develop` after https://github.com/Chia-Network/clvm_tools/pull/26lands.